test/e2e/traceflow_test.go

```diff
@@ -127,6 +127,7 @@ func TestTraceflow(t *testing.T) {
 						TransportHeader: v1alpha1.TransportHeader{
 							TCP: &v1alpha1.TCPHeader{
 								DstPort: 80,
+								Flags:   2,
```

Comment:
@lzhecheng a bit of a late question, but why this change. Isn't the SYN flag only required for Pod-to-Service Traceflow?

Reply:
Yes, it was added by mistake. I think we can remove it when other related PR is made since it doesn't make trouble here?

Reply:
No worries, I was just curious. Do you have an answer for the UI question below?